iocore/net/UnixNetVConnection.cc

```diff
@@ -1514,9 +1514,10 @@ UnixNetVConnection::set_inactivity_timeout(ink_hrtime timeout_in)
 #else
   if (timeout_in == 0) {
```

Comment:
Interesting! Thanks for describing the inactivity cop behaviors. I understood my proposal doesn't make sense.

BTW, #771 is reverted in 7.1.x branch, because of performance issue  (see https://github.com/apache/trafficserver/commit/db596a1c63e1cc86cfd066ed08134750dabe7d66). Do we still need separate PR for 7.1.x?